src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -88837,7 +88891,7 @@ static class E
 }
 class Program
 {
-    static void F(Pair<object, object?>? p)
+    static void F(Pair<object, object?> p)
```

Comment:
>F(Pair<object, object?> p) [](start = 16, length = 26)

Please revert this change to ensure we're not reporting a warning for invocation of `Deconstruct`.